dropwizard-health/src/main/java/io/dropwizard/health/response/HealthResponse.java

```diff
@@ -1,57 +1,67 @@
 package io.dropwizard.health.response;
 
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
+import io.dropwizard.health.HealthStateView;
+import java.util.Collection;
 import java.util.Objects;
-import java.util.Optional;
+import javax.annotation.Nonnull;
 
-public abstract class HealthResponse {
-    protected final boolean healthy;
-    @Nullable
-    protected final String message;
+public final class HealthResponse {
+    private final boolean healthy;
```

Comment:
Maybe we should move the logic of what the response code is into the HealthResponseProvider? That would more easily open the door if someone wanted to try and make the health response non-binary (healthy vs unhealthy).

Reply:
Great suggestion -- I think a boolean is unnecessarily inflexible here

Reply:
Or at least, **only** a boolean. I'll throw in a status code additionally.